app/scripts/authentication/authentication.config.js

```diff
@@ -0,0 +1,10 @@
+(function () {
+    'use strict';
+
+    var authModule = angular.module('ngAmrsApp');
+    
+    //add authorization interceptor
+    authModule.config(['$httpProvider', function ($httpProvider) {
+        $httpProvider.interceptors.push('authorizationInterceptor');
```

Comment:
you don't need a new file for this. You can actually put this one line right after [this line](https://github.com/AMPATH/ng-amrs/blob/master/app/scripts/app.module.js#L396)